before-run.sql

```diff
@@ -3,7 +3,7 @@ alter system set `planner.enable_decimal_data_type` = true;
 alter system set `new_view_default_permissions` = '777';
 alter system set `planner.enable_limit0_optimization` = true;
 alter system set `exec.errors.verbose` = true;
-alter system set `planner.memory.max_query_memory_per_node` = 10737418240;
+alter system set `planner.memory.max_query_memory_per_node` = 7737418240;
```

Comment:
Any reason to reduce this? On the regression clusters, we assume 32GB of direct memory. So lets keep this 10GB. In fact might need to increase this to 12 later.